types/fetch-mock/index.d.ts

```diff
@@ -165,23 +166,23 @@ declare namespace fetchMock {
     }
 
     interface MockOptionsMethodGet extends MockOptions {
-    method: 'GET';
+        method?: never;
```

Comment:
This looks like it might break callers who specify `method`.  Is there a corresponding change to the version number of the library?  If not, would it be preferable to keep the type the same and make the property optional so that new callers don't have to specify it?

Reply:
> Is there a corresponding change to the version number of the library?

No. Previous type definition is wrong.

> If not, would it be preferable to keep the type the same and make the property optional so that new callers don't have to specify it?

OK. I'll do that